cdap-cli/src/main/java/co/cask/cdap/cli/command/GetWorkflowLocalDatasetsCommand.java

```diff
@@ -59,7 +59,7 @@ public void perform(Arguments arguments, PrintStream printStream) throws Excepti
       throw new CommandInputError(this);
     }
 
-    ProgramRunId programRunId = new ProgramRunId(cliConfig.getCurrentNamespace().getId(), programIdParts[0],
+    ProgramRunId programRunId = new ProgramRunId(cliConfig.getCurrentNamespace().getNamespace(), programIdParts[0],
```

Comment:
will ``cliConfig.getCurrentNamespace().app(programIdParts[0]).workflow(programIdParts[1]).run(arguments.get(ArgumentName.RUN_ID.toString())`` be better?